Prox/Prox/PlaceCarouselViewController.swift

```diff
@@ -218,7 +226,7 @@ extension PlaceCarouselViewController: CLLocationManagerDelegate {
         mapView.region = MKCoordinateRegion(center: center, span: span)
 
         FirebasePlacesDatabase().getPlaces(forLocation: location).upon(DispatchQueue.main) { places in
-            self.placeCarousel.places = places.flatMap { $0.successResult() }
+            self.places = places.flatMap { $0.successResult() }
```

Comment:
I agree that your first option is better, but instead of using a closure, we should create a delegate protocol to provide the data to the `PlaceCarousel`. That same delegate can then be used to provide data to the place details screens that we are using in https://github.com/mozilla-mobile/prox/tree/fluffyemily/issue-43-open-detail-view-from-place-carousel